vue-components/stories/Dropdown.stories.ts

```diff
@@ -80,6 +80,68 @@ export function basic( args ): Component {
 	};
 }
 
+export function all( ): Component {
+	return {
+		components: { Dropdown },
+		data(): unknown {
+			return {
+				menuItems:menuItems,
+				label: 'Label',
+				selectedItem: null,
+			};
+		},
+		computed: {
+			error(): any {
+				if ( this.selectedItem?.value === 'later' ) {
+					return { type: 'warning', message: 'Warning to be careful 🚧' };
+				}
+				if ( this.selectedItem?.value === 'earlier' ) {
+					return { type: 'error', message: 'There was an error 😕' };
+				}
+				return null;
+			},
+		},
+		template: `
+			<div><div style="max-width: 512px">
+				<Dropdown
+					:label="label"
+					:menu-items="menuItems"
+					v-model="selectedItem"
+					placeholder="Select an option"
+					:disabled="disabled"
+					:error="error"
+				/>
+
+				<p><Dropdown
+				:label="label"
+				:menu-items="menuItems"
+				v-model="selectedItem"
+				placeholder="Select an option"
+				:disabled="true"
```

Comment:
Chromatic is complaining about "disabled" being undefined. I think it could be related to this line, the problem persists if you define > `:disabled="true"`, `disabled="disabled"` or `disabled="true"`. All those options make the dropdown component appear correctly disabled in localhost:6005.